src/HypothesisTests.jl

```diff
@@ -122,11 +123,11 @@ function Base.show(io::IO, test::T) where T<:HypothesisTest
     println(io, "Test summary:")
     println(io, "    outcome with 95% confidence: $outcome h_0")
     if tail == :both
-        println(io, "    two-sided p-value:           $p")
+        println(io, "    two-sided p-value:           $(StatsBase.PValue(p))")
```

Comment:
Here too it would make sense to store the `PValue` in an object so that you don't need to call this for each case.

Reply:
I have a more fundamental question. Couldn't we, as has been done for the confidence interval (see https://github.com/JuliaStats/HypothesisTests.jl/issues/67), use StatsBase.PValue throughout and extend it for our tests?

Reply:
Well, `PValue` is mostly useful for printing AFAIK. It could be annoying to return such an object to users, as it doesn't support any operation defined on numbers.

Reply:
I just realised this as well.